crates/tuono/src/route.rs

```diff
@@ -186,6 +186,19 @@ mod tests {
         assert_eq!(dyn_info.module_import, "dyn_posts");
     }
 
+    #[test]
+    fn should_handle_hyphen() {
```

Comment:
Thanks for the test. Right now we have already a test that check for the correct route parsing. in `src/app.rs`. The test name is `should_collect_routes`. Could you please just add the map there? I agree that the test should better apply for the `route.rs` file but I'd keep all together for now